### ISSUE TITLE
[BiliBili] Unable to download JSON metadata

### ISSUE THREAD

**Issue by brian6932** [yt-dlp/yt-dlp#5450]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.10.04** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

United States

### Provide a description that is worded well enough to be understood

I don't know anymore than the debug logs

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
❯ yt-dlp -Uv https://www.bilibili.com/video/BV1LK4y1v7ke/
[debug] Command-line config: ['-Uv', 'https://www.bilibili.com/video/BV1LK4y1v7ke/']
[debug] Portable config "~\scoop\apps\yt-dlp\current\yt-dlp.conf": []
[debug] Encodings: locale cp65001, fs utf-8, pref cp65001, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.10.04 [4e0511f] (win32_exe)
[debug] Python 3.8.10 (CPython 64bit) - Windows-10-10.0.20152-SP0
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg N-108915-g5661c8715c-20221102 (setts), ffprobe N-108915-g5661c8715c-20221102
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.09.24, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] Loaded 1690 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.10.04, Current version: 2022.10.04
yt-dlp is up to date (2022.10.04)
[debug] [BiliBili] Extracting URL: https://www.bilibili.com/video/BV1LK4y1v7ke/
[BiliBili] 1LK4y1v7ke: Grabbing original ID via API
ERROR: [BiliBili] Unable to download JSON metadata: <urlopen error [Errno 11004] getaddrinfo failed> (caused by URLError(gaierror(11004, 'getaddrinfo failed')))
  File "yt_dlp\extractor\common.py", line 672, in extract
  File "yt_dlp\extractor\bilibili.py", line 161, in _real_extract
  File "yt_dlp\extractor\bilibili.py", line 406, in _get_video_id_set
  File "yt_dlp\extractor\common.py", line 1032, in download_content
  File "yt_dlp\extractor\common.py", line 996, in download_handle
  File "yt_dlp\extractor\common.py", line 866, in _download_webpage_handle
  File "yt_dlp\extractor\common.py", line 823, in _request_webpage

  File "urllib\request.py", line 1354, in do_open
  File "http\client.py", line 1252, in request
  File "http\client.py", line 1298, in _send_request
  File "http\client.py", line 1247, in endheaders
  File "http\client.py", line 1007, in _send_output
  File "http\client.py", line 947, in send
  File "http\client.py", line 918, in connect
  File "socket.py", line 787, in create_connection
  File "socket.py", line 918, in getaddrinfo
socket.gaierror: [Errno 11004] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "yt_dlp\extractor\common.py", line 805, in _request_webpage
  File "yt_dlp\YoutubeDL.py", line 3682, in urlopen
  File "urllib\request.py", line 525, in open
  File "urllib\request.py", line 542, in _open
  File "urllib\request.py", line 502, in _call_chain
  File "yt_dlp\utils.py", line 1334, in http_open
  File "urllib\request.py", line 1357, in do_open
urllib.error.URLError: <urlopen error [Errno 11004] getaddrinfo failed>
```

**Comment by PikuZheng** [comment]
I had the same problem. It was worked about 1 month ago.

**Comment by bashonly** [comment]
This issue has been fixed in master (ad97487606c87878aa06b736a72ffde15056bdd4, c90c5b9bddfaa36afd07db676e351571fce102e8). Please wait for the next release, [use the master branch](https://github.com/yt-dlp/yt-dlp/wiki/Installation#with-pip), or try the next release of the [unofficial daily builds](https://github.com/ytdl-patched/yt-dlp)

**Comment by Lesmiscore** [comment]
[The latest daily build](https://github.com/ytdl-patched/yt-dlp/releases/tag/2022.11.06.334) should contain these commits
